lib/src/configuration.cc

```diff
@@ -156,6 +161,11 @@ HW::ParseResult Configuration::parseOptions(int argc, char *argv[])
     // This is primarily for testing.
     master_uris_.clear();
 
+    // remember the path to the pxp-agent executable used to start
+    // this process, it is supposed to be used to start executables
+    // which are installed alongside the pxp-agent executable
+    exec_prefix_ = fs::absolute(fs::path(argv[0]).parent_path());
```

Comment:
Ah; I thought I encountered a problem using a non-absolute path and trying to run tasks. I'll retry it to check.